acceptance/detector_test.go

```diff
@@ -193,4 +199,125 @@ func testDetector(t *testing.T, when spec.G, it spec.S) {
 			h.AssertStringContains(t, logs, expectedAppDir)
 		})
 	})
+
+	when("-order is not provided directly", func() {
```

Comment:
What do you think about the following order under this "when":

when("platform api < 0.6")
{
    when ("/cnb/order.toml and /layers/order.toml are present") {...}

    when(“only /layers/order.toml is present”) // to add - there will be an error, right? {...}
}

when("/cnb/order.toml and /layers/order.toml are present") {...}

when(“only /cnb/order.toml” is present”) {...}

when(“only /layers/order.toml is present”) {...}

I would also add a test where `-order` is provided.